site/profile/manifests/core/telegraf.pp

```diff
@@ -14,13 +14,14 @@
 #   The backing InfluxDB instance. At present the default DNS record is a CNAME.
 class profile::core::telegraf(
   String $password,
-  String $username = 'telegraf-nodes',
-  String $database = 'nodes',
+  String $username = 'telegraf',
+  String $database = 'telegraf',
   String $host     = 'metrics-ingress.ls.lsst.org',
 ) {
   class { '::telegraf':
     hostname    => $::facts['fqdn'],
     global_tags => {'site' => $::site},
+    outputs     => {},  # suppress class defaults, it populates an empty influx instance
```

Comment:
It might be more reasonable to use this output rather than the separate `telegraf::output` defined type.